src/app/components/Image/index.stories.tsx

```diff
@@ -42,32 +42,50 @@ export const ResponsiveWebPWithJpegFallback = () => (
 );
 
 export const BasicAMPImage = () => (
-  <Image
+  <RequestContextProvider
     isAmp
-    alt="A penguin stands on an ice floe"
-    src="https://ichef.bbci.co.uk/ace/ws/624/cpsprodpb/164AF/production/_110911319_antartica.jpg"
-  />
+    pageType="article"
+    pathname="/afaanoromoo"
+    service="afaanoromoo"
+  >
+    <Image
+      alt="A penguin stands on an ice floe"
+      src="https://ichef.bbci.co.uk/ace/ws/624/cpsprodpb/164AF/production/_110911319_antartica.jpg"
+    />
+  </RequestContextProvider>
 );
```

Comment:
Why don't we create a wzrapper `Component`, which already has the `RequestContextProvider` with default values, to avoid using having to include it in every story: we do a similar thing here: https://github.com/bbc/simorgh/blob/latest/src/app/components/Caption/index.stories.tsx - you could then pass in just the values you need to override

Reply:
We could even hardcode the image data, as it seems to be the same for every story